security/user.go

```diff
@@ -192,3 +188,31 @@ func (u *User) persist(action string, body interface{}, options types.QueryOptio
 
 	return u, nil
 }
+
+func (u *User) queryCredentials(action string, strategy string, body interface{}, options types.QueryOptions) (json.RawMessage, error) {
```

Comment:
So there is no way to call create *MyCredentials methods ?
And no way to call *Credentials without instantiating a user first ?

Reply:
@dbengsch  > About your second point, I don't know of a use case where it would be useful to change a user's credentials without instantiating a `User` object first. Do you have one?

Reply:
@scottinet As the overhead to instantiate a new User is minimal, comprehension is better. I was just pointing it to be sure.

Reply:
ragarding *MyCredentials, there are still available in kuzzle container.

Reply:
oh, okay then